README.md

```diff
@@ -16,42 +16,71 @@ The SAF CLI is the successor to [Heimdall Tools](https://github.com/mitre/heimda
-  * [Generate](#generate) - Generate InSpec validation code, set pipeline thresholds, and generate options to support other saf commands.
-  * Scan - Visit https://saf.mitre.org/#/validate to explore and run inspec profiles
-  * Harden - Visit https://saf.mitre.org/#/harden to explore and run hardening scripts
+- [Security Automation Framework CLI](#security-automation-framework-cli)
+  - [Terminology:](#terminology)
+  - [Contents:](#contents)
+  - [Installation](#installation)
+      - [Installation via NPM](#installation-via-npm)
+      - [Update via NPM](#update-via-npm)
+      - [Installation via Docker](#installation-via-docker)
+      - [Update via Docker](#update-via-docker)
+      - [Installation via Windows Installer](#installation-via-windows-installer)
+      - [Update via Windows Installer](#update-via-windows-installer)
+  - [## Usage](#-usage)
+    - [Attest](#attest)
+      - [Create Attestations](#create-attestations)
+      - [Apply Attestations](#apply-attestations)
+    - [Convert](#convert)
+    - [From HDF](#from-hdf)
+        - [HDF to ASFF](#hdf-to-asff)
+      - [HDF to Splunk](#hdf-to-splunk)
+      - [HDF to XCCDF](#hdf-to-xccdf)
+        - [Previewing HDF Data Within Splunk:](#previewing-hdf-data-within-splunk)
+        - [HDF to Checklist](#hdf-to-checklist)
+        - [HDF to CSV](#hdf-to-csv)
+        - [HDF to Condensed JSON](#hdf-to-condensed-json)
+    - [To HDF](#to-hdf)
+        - [ASFF to HDF](#asff-to-hdf)
+        - [AWS Config to HDF](#aws-config-to-hdf)
+        - [Burp Suite to HDF](#burp-suite-to-hdf)
+        - [CKL to POA&M](#ckl-to-poam)
+        - [DBProtect to HDF](#dbprotect-to-hdf)
+        - [Fortify to HDF](#fortify-to-hdf)
+        - [Twistlock to HDF](#twistlock-to-hdf)
+        - [JFrog Xray to HDF](#jfrog-xray-to-hdf)
+        - [Ion Channel 2 HDF](#ion-channel-2-hdf)
+        - [Tenable Nessus to HDF](#tenable-nessus-to-hdf)
+        - [Netsparker to HDF](#netsparker-to-hdf)
+        - [Nikto to HDF](#nikto-to-hdf)
+        - [Prisma to HDF](#prisma-to-hdf)
+        - [Prowler to HDF](#prowler-to-hdf)
+        - [Sarif to HDF](#sarif-to-hdf)
+        - [Scoutsuite to HDF](#scoutsuite-to-hdf)
+        - [Snyk to HDF](#snyk-to-hdf)
+        - [SonarQube to HDF](#sonarqube-to-hdf)
+        - [Splunk to HDF](#splunk-to-hdf)
+        - [Trivy to HDF](#trivy-to-hdf)
+        - [XCCDF Results to HDF](#xccdf-results-to-hdf)
+        - [OWASP ZAP to HDF](#owasp-zap-to-hdf)
+    - [View](#view)
+      - [Heimdall](#heimdall)
+      - [Summary](#summary)
+    - [Validate](#validate)
+      - [Thresholds](#thresholds)
+    - [Generate](#generate)
+      - [CKL Templates](#ckl-templates)
+      - [InSpec Metadata](#inspec-metadata)
+      - [Thresholds](#thresholds-1)
+      - [Spreadsheet (csv/xlsx) to InSpec](#spreadsheet-csvxlsx-to-inspec)
+      - [XCCDF to InSpec Stub](#xccdf-to-inspec-stub)
+      - [Other](#other)
+        - [Notes](#notes)
+        - [Mapping Files](#mapping-files)
```

Comment:
![hmm](https://user-images.githubusercontent.com/66680985/178525698-01a16f22-f1e0-4b2b-a2dc-971045ece3af.png)
There seems to be some markdown tags being shown inside of the rendered text. Also please confirm with Eugene if this layout is ok if you haven't

Reply:
Eugene made some changes and gave his approval, we should be good on the ToC changes.